gazebo_plugins/test/worlds/gazebo_ros_ackermann_drive.world

```diff
@@ -540,6 +540,7 @@
           <remapping>cmd_vel:=cmd_test</remapping>
           <remapping>odom:=odom_test</remapping>
           <remapping>distance:=distance_test</remapping>
+          <remapping>steerangle:=steerangle_test</remapping>
```

Comment:
I can see here that you added the `steerangle` to the test world, do you mind to add a test to the current `ackermann_drive` tests ?

Reply:
I added test code to ackermann_drive tests. The changes are below.